Packages/vcs/Lib/projection.py

```diff
@@ -21,6 +21,9 @@
 round_projections = ['polar (non gctp)', 'stereographic',
                      'orthographic', "ortho", ]
 
+no_over_proj4_parameter_projections = round_projections+["aeqd", "lambert conformal c"]
```

Comment:
shouldn't be, because polar do not go to the part of the code that uses `no_over_proj4_parameter_projections` I will take a look at the animation test now.